templates/base.html

```diff
@@ -5,10 +5,6 @@
 </head>
 <body class="html front not-logged-in one-sidebar sidebar-first page-node " >
 {% include 'includes/top-hat.html' %}
-<!--  I'm not sure what this is here for
```

Comment:
It's good practice if you have header, slideshow, etc at the top of your page and content below - this allows people using screen readers or other assistive technology to skip all the stuff they can't see anyway and get to the content. It requires a #content anchor where the content begins.